### PR TITLE
Add tests

### LINKED ISSUE #2: Add tests
Going to work on this in https://github.com/opdavies/tailwindcss-flexbox-order.

### PR DESCRIPTION
**Pull request by opdavies**
Fixes #2

### CODE REVIEW

**Comment by opdavies** [comment]
Run with `jest` or `npm test`.

**Comment by adevade** [comment]
Thank you very much!

**Comment by opdavies** [comment]
@adevade You're welcome. 😄

File: test.js [hunk 1]
```diff
@@ -1,249 +1,222 @@
...
+        .then(result => {
+            return result.css;
+        });
+}
 
 const disableCorePlugins = () => {
-  return _.mapValues(defaultConfig.variants, plugin => {
-    return false
-  })
+    return _.mapValues(defaultConfig.variants, () => false);
```
Comment (opdavies) [inline test.js:26]: This is better. Thanks for improving it! 😄

**Pull request merged**